parity/cli/mod.rs

```diff
@@ -211,6 +211,8 @@ usage! {
 			or |c: &Config| otry!(c.footprint).cache_size_blocks.clone(),
 		flag_cache_size_queue: u32 = 50u32,
 			or |c: &Config| otry!(c.footprint).cache_size_queue.clone(),
+		flag_cache_size_state: u32 = 25u32,
```

Comment:
I don't agree. 64MB is quite large and I find that there are diminishing returns after a certain point. With the default size, I find that the heavy blocks only process twice as slowly as with an unrestricted cache.

A priority should be placed on keeping the footprint light by default. Miners have both the resources and the technical knowledge to increase the cache size using the flag.